pkg/destroy/powervs/powervs.go

```diff
@@ -752,14 +757,19 @@ func (o *ClusterUninstaller) ServiceInstanceNameToGUID(ctx context.Context, name
 			}
 
 			if resourceInstance.Type == nil || resourceInstance.GUID == nil {
+				o.Logger.Debugf("ServiceInstanceNameToGUID: type: nil")
```

Comment:
Not necessarily the `type` is `nil` here since it can also happen that only `GUID` is `nil`. Maybe 
```suggestion
				o.Logger.Debugf("ServiceInstanceNameToGUID: type: %v", resourceInstance.Type)
```
to be accurate?

Reply:
Good point!  How about this?